### ISSUE TITLE
Story format search directories not found after building from source

### ISSUE THREAD

**Issue by Cllezark** [tmedwards/tweego#15]
I am setting up work environments for Go and Tweego for the first time on Ubuntu 20.04 and macOS 10.15.x. After following the directions given to build Tweego from source, calling `tweego` with any command causes this to appears in my console:
 

> `error: Story format search directories not found.` 

Calling `tweego -help` returns the same error. Where could I have gone wrong?

**Comment by tmedwards** [comment]
Story formats are not part of the source repository, so if you're building Tweego manually, rather than using one of the pre-built releases, then you need to provide them yourself.

An easy way to get the basic set would be to download [the latest Tweego release](https://github.com/tmedwards/tweego/releases/latest) and copy the `storyformats` directory, and its contents, from it—it may not contain the latest version of all included story formats, but it should get you started.  Alternatively, if you don't feel the need for the basic Twine 2 set, then just download the latest version of whatever story format you want from its website.

The [*Story Formats* section](http://www.motoslave.net/tweego/docs/#getting-started-story-formats) of Tweego's documentation explains how to allow the Tweego binary to find your story formats.

**Comment by Cllezark** [comment]
Thanks for the quick response! I'm partly concerned that I may have stumbled into a build error; even `tweego --help` is blocked by the error flag. Is that intentional?
